ProjectTemplates/Projects/App-Camera/Pages/Lightbox/LightBoxPage.js

```diff
@@ -0,0 +1,14 @@
+var App = require("_$filename$_");
+var images = App.getPictures();
+module.exports = {
+	images : images,
+	currentIndex : this.Parameter.map(function(param){
+		var idx = images.length;
```

Comment:
Yeah it was used for a reverse while over the images at one point, removed now :)